en/components/bullet-graph.md

```diff
@@ -12,100 +12,62 @@ For scenarios where no target value exists or is irrelevant you should use the [
 
 ## Bullet Graph Demo
 
-<img class="responsive-img" src="../images/bullet_graph_three_ranges.png" srcset="../images/bullet_graph_three_ranges@2x.png 2x" />
-
-## Ranges Amount
-
-The Bullet Graph comes with three preset range configurations: two, **three**, and four ranges.
-
-<img class="responsive-img" src="../images/bullet_graph_two_ranges.png" srcset="../images/bullet_graph_two_ranges@2x.png 2x" />
-<img class="responsive-img" src="../images/bullet_graph_three_ranges.png" srcset="../images/bullet_graph_three_ranges@2x.png 2x" />
-<img class="responsive-img" src="../images/bullet_graph_four_ranges.png" srcset="../images/bullet_graph_four_ranges@2x.png 2x" />
-
-## Styling
-
-The Bullet Graph comes with styling flexibility through the various overrides controlling the background color and a range of brushes including: Value Brush, Target Value Brush, Tick Brush, Minor Tick Brush, and a collection of Range Brushes.
-
-<img class="responsive-img" src="../images/bullet_graph_styling.png" srcset="../images/bullet_graph_styling@2x.png 2x" />
-
-## Usage
-
-When creating data visualizations and information dashboards pick carefully between Bullet Graph and Linear Gauge depending on whether your scenario has a target value or not. Keep in mind that both Graphs and Gauges only show the current value and have no way to display historical values, if your application scenario requires such display, you should consider using the [Category Chart](chart-category.md). To create an easy to the eye and crisp data visualization pick colors for the Value and Target Value Brushes that have good contrast with the ranges and background underneath, avoid using variants of the same color as this makes the visualization hard to read and interpret.
-
-| Do                                                                                       | Don't                                                                                        |
-| ---------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
-| <img class="responsive-img" src="../images/bullet_graph_do.png" srcset="../images/bullet_graph_do@2x.png 2x" /> | <img class="responsive-img" src="../images/bullet_graph_dont.png" srcset="../images/bullet_graph_dont@2x.png 2x" /> |
-
-## Code Generation
+<img class="responsive-img" src="../images/bullet_graph_three_ranges-2.png" srcset="../images/bullet_graph_three_ranges@2x-2.png 2x" />
```

Comment:
```suggestion
<img class="responsive-img" src="../images/bullet_graph_three_ranges-2.png" srcset="../images/bullet_graph_three_ranges@2x.png 2x" />
```

Reply:
Isn't it better to use Infragistics' name in all the sample images instead of Facebook

Reply:
This is the example for which we have the data. I don't know what the revenue of Infragistics is, not I think it's ok to put it out there. The Facebook's is accessible online by everyone. Also I haven't seen any of our competitors use their name in their samples, so that's why I made it with Facebook, even though it's a sample also used in the UI Kit.

Reply:
Yes let's keep it as is, I will close this comment